### DISCUSSION TITLE
Using subsets

### DISCUSSION THREAD

**Discussion by guillemc23** [sertit/eoreader#185] [Q&A] [answered]
I'm still new to this library and SNAP, but since these processes are usually quite intensive, wouldn't it be a great idea to add by default a subset operation? In case some WKT is provided, clip the result to it and in case it is not, just dont clip it 🤔 

I didn't see any parameter to specify this subset in the FAQ: https://eoreader.readthedocs.io/en/latest/sar.html#what-to-know-if-you-are-changing-a-graph

**Comment by remi-braun** [comment, marked as answer]
You're right, for now it isn't handled. I'll create an issue, because this usecase is legit.

I have to raise some points however:
- I plan to use the `window` keyword as for other sub-setting operations
- asking the footprint of the products will still orthorectify the whole product
- loading another window of your tile will redo the orthorectification over this new window

Do you agree?

**Reply by remi-braun** [reply]
See https://github.com/sertit/eoreader/issues/186

**Reply by guillemc23** [reply]
Not an expert in GPT graphs, but if the clipping is done as soon as possible, it will definitely reduce processing time 😄 all of the points you mention sound fine to me

**Reply by remi-braun** [reply]
I hope I'd have time soon to update EOReader, there is a lot of opened issues at the moment 😅

**Reply by guillemc23** [reply]
I see, I'd help if I saw something I can solve, but I'm still very new to this processes 😅 will keep testing and trying with products and reporting issues, but still a very nice library!

**Reply by remi-braun** [reply]
What you do already is extremely valuable! Thanks for taking the time to report issues/open discussions. All are very pertinent to my eyes 🙏
